### DISCUSSION TITLE
Shortcut for SkinEditor

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#20194] [Ideas] [answered]
https://user-images.githubusercontent.com/34775378/189129309-aff73a9a-130f-41d8-afc9-38b46a70d571.mp4

like in osu edior, this makes it easier for the user to make fine adjustments

**Comment by peppy** [comment]
Is there a reason it can't just be arrows without ctrl being held?

**Reply by cdwcgt** [reply]
I think there should be no need to hold down ctrl in this situation
because I copied the logic from std blueprint, just remove if (e.ControlPressed) and it can work

**Reply by nagi-desuuu** [reply]
Speaking from experience, using arrows to "nudge" UI elements is an expected UX flow.

**Reply by cdwcgt** [reply]
yes I have changed the logic, no need to hold ctrl to move

And this is also a small change, can open a pr for it?

**Reply by peppy** [reply]
Go for it.

**Comment by cdwcgt** [comment, marked as answer]
https://github.com/ppy/osu/pull/20195
